atc/db/team_test.go

```diff
@@ -1832,8 +1830,7 @@ var _ = Describe("Team", func() {
 			resource, found, err = savedPipeline.Resource("some-resource")
 			Expect(err).ToNot(HaveOccurred())
 			Expect(found).To(BeTrue())
-			Expect(resource.ConfigPinnedVersion()).To(BeNil())
-			Expect(resource.APIPinnedVersion()).To(BeNil())
```

Comment:
I think that the `ConfigPinnedVersion` and `APIPinnedVersion` are being used as a proxy for what is in the database rows here. This test is still valid I think if you trust that `resource.PinnedVersion()` will return non-nil if the version was pinned either by API or config (which is unit tested so should hold)